<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1911</width>
    <height>1064</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Сучков_ВН_2уч</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>238</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Точность (контроль погрешности):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_precision">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Размер шага метода:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_step">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>245</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_x">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Участок интегрирования:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_x_start">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_x_end">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>262</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_start">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Начальные условия:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_start_x">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_start_y">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_last">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>263</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_mistake">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Контроль локальной погрешности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_mistake">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>701</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>870</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Цвет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_blue">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Синий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_red">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Красный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_green">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Зеленый</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_violet">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Фиолетовый</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="getdata_buttom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Запустить просчет</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)
</string>
    </property>
    <item>
     <property name="text">
      <string>Задание тест</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Задание 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Основное задание</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>911</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_n_2">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_a">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_n_3">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_b">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_n_4">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_c">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_n">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>N :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_n">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>189</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Количество шагов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Параметры:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>870</y>
      <width>229</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_table">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>870</y>
      <width>229</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Заполнить таблицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>900</y>
      <width>229</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plot_from_file">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>930</y>
      <width>229</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Построить из файла ( - )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_points">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>900</y>
      <width>229</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;FiraCode Nerd Font Propo&quot;;
color:rgb(0, 0, 0);
color-bg:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Сохранить в файл ( - )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>940</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Помощь</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс. шаг:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин. шаг:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс. ЛП:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс. Глоб. Погр:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Число шагов метода:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="step_num_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>810</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="max_glob_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>760</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="max_le_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>710</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="min_h_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>660</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="max_h_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>610</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1911</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
